opera-developer/PKGBUILD

```diff
@@ -39,6 +39,6 @@ package() {
 	
 	unlink ${pkgdir}/usr/bin/${pkgname}
 	cd ${pkgdir}/usr/bin
-	ln -sr ../usr/lib/opera-developer/opera-developer
+	ln -sf /usr/lib/${pkgname}/${pkgname}
 }
```

Comment:
Okay I will check it out. The last couple releases I havent been able to build it because the source url was returning a 404 Forbidden error. Though it was only doing it when accessed from our build server. As soon as I have some time I will check on it.